src/containers/Articles/Article/RatingContainer.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import Rating from '../../../components/Rating';
+import { rateFetch, updateRateValue } from '../../../store/actions/async/rate';
+
+
+export class RatingContainer extends Component {
+    state = {
+        rate_score: 0
+    }
+    componentDidMount = () => {
+
+        const { slug } = this.props.match.params;
```

Comment:
Destructuring nested components: `const { match: {params: {slug}} = {params: {}} } = this.props;`.  This gives us the `slug`.
We can also have a `slug` in the `state` and we just update it in `componentDidMount` so that we don't duplicate code on line `30`